system_tests/staker_test.go

```diff
@@ -51,6 +58,29 @@ func makeBackgroundTxs(ctx context.Context, l2info *BlockchainTestInfo, l2client
 	return nil
 }
 
+func validatorDataposter(db ethdb.Database, l1Reader *headerreader.HeaderReader,
```

Comment:
nit. can't we just make the validatorDataposter in node.go public and call it here?

Reply:
Done.